src/defaultParseAndHandleMessage.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+// @flow
+
+import type { MessageHandler, MessageResultHandler, MessageContentHandler, JsonValue } from './index'
+
+import parseAndHandleMessage from './parseAndHandleMessage'
+import parseJsonMessage from './parseJsonMessage'
+
+export default (handleMessage: MessageContentHandler<JsonValue, mixed>): MessageHandler => {
+  // Generally, errors won't be recovered from
+  // So default behavior here is always to ack, thus dropping failed messages,
+  // rather than nacking, which can lead to infinite retries if not managed carefully
+  const ackOnComplete: MessageResultHandler<mixed> = (ack, nack, message) => ack(message)
+  return parseAndHandleMessage(parseJsonMessage('utf8'), ackOnComplete, ackOnComplete, handleMessage)
```

Comment:
I don't like that `'utf8'` is hardcoded in two places in this package, but I'm willing just to go with it and open an issue to "fix" it later.